pkg/template/template.go

```diff
@@ -879,3 +883,18 @@ func splitListTrim(sep, orig string) []string {
 	processedStr := strings.TrimRight(strings.TrimLeft(orig, sep), sep)
 	return strings.Split(processedStr, sep)
 }
+
+// For given []string as input, randomize API returns its randomized list
+// Example:
+// {{- $poolUids := keys .ListItems.cvolPoolList.pools | randomize }}
+func randomize(list []string) []string {
+	res := []string{}
+
+	r := rand.New(rand.NewSource(time.Now().Unix()))
+	perm := r.Perm(len(list))
+	for _, idx := range perm {
```

Comment:
`idx` does not look idiomatic.

Reply:
any suggestions please?

Reply:
As I can understand, perm will is slice of randomized indexes.
Following are some words that are coming to my mind:
`key` , `index` 
I would prefer to go with `key` .  (As it tells me that this is key of some value). Your call.